keysign/discover.py

```diff
@@ -42,7 +42,11 @@ def start(self):
         if self.bt_code and not self.stopped:
             # We try Bluetooth, if we have it
             log.info("Trying to connect to %s with Bluetooth", self.bt_code)
-            self.bt = BluetoothReceive(self.bt_port)
+            if self.bt is not None:
```

Comment:
Why are you checking against `self.bt`? When the class is initialized `self.bt` is None, so this `if` will never pass.